docs/getting-started.md

```diff
@@ -187,11 +191,11 @@ print(answers)
 * Line 29: Select the value of `True` for the question asking the user if they like dogs.
 
 !!! note
-    The `arg` parameter to `get_answers()` can be omitted to use the values in `sys.argv`.
+    Omitting the `args` parameter to `parse_args()` will use values in `sys.argv`.
```

Comment:
This might be a good time to start moving the docs towards active voice 🏃 :

```suggestion
    If you omit the `args` parameter to `parse_args()` the values in `sys.argv` will be used.
```

Ref: https://www.grammarly.com/blog/passive-voice/